### DISCUSSION TITLE
Saving and selecting color schemes

### DISCUSSION THREAD

**Discussion by TheLarry98** [OpenRCT2/OpenRCT2#15754] [Feature requests]
Some rides like the Spiral Slide Or Merry-Go-Round will have a different color when building them.
It would be great when we would be able to choose predefinied color combinations in the color tab.
And also be able to save our own color combinations

Right now I select the Spiral Slide until I have the color combination I want or I have to change the colors manually in the color tab.

It could look like this:
![openrct2 color combinations](https://user-images.githubusercontent.com/12634026/118147749-31a05680-b410-11eb-9324-1db6d914f7d0.png) (left ride window old, right ride window new)

**Comment by Gymnasiast** [comment]
I suppose we should take a look at the predefined colour schemes in general. Some are really ugly, like the black-with-light blue for the Wooden RC, or the red-with-orange scheme for the Twister RC.

**Comment by ocalhoun6** [comment]
Maybe this would be more of a plugin kind of thing?

**Comment by Ryder17z** [comment]
> I suppose we should take a look at the predefined colour schemes in general. Some are really ugly, like the black-with-light blue for the Wooden RC, or the red-with-orange scheme for the Twister RC.

that's actually one of the better ones i think  :stuck_out_tongue:

**Comment by Broxzier** [comment]
Perhaps having the default ones as standard presets, and allowing players to edit it all, so default ones they dislike can be removed too.

**Comment by Ryder17z** [comment]
> Perhaps having the default ones as standard presets, and allowing players to edit it all, so default ones they dislike can be removed too.

We already have a theme editor, so yes, this would be nice. E.g. I'd like to disable the red/green preset for Enterprise and light-blue for the merry-go-round
